health/adr/default_dependencies_primary.md

```diff
@@ -0,0 +1,18 @@
+## **Title**: Dependencies Default to Primary
```

Comment:
I want to check on @nathanschile's opinion. Is this the desired structure for ADRs or should there be one adr directory at the root of the project and then the filenames/titles could provide the additional context for which module they relate too?

Reply:
I think we should have the adr directory at the root. This would help with global decisions that may are not limited to a particular module.

Reply:
I moved the adr directory to root, I wasn't sure what directory structure to have, so I just put the adr under the `adr/health` directory. Do we want something like that? Or I was thinking about breaking it up by concept, in this case, having a `adr/dependency` directory. Opinions?

Commit: https://github.com/cerner/beadledom/pull/123/commits/a44583a8072c278fdc538d566510516b2cbb8f23

@johnlcox @nathanschile